src/driver-holtek.h

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+/*
+ * Copyright © 2023 Michał Lubas
+ */
+
+#pragma once
+
+#include "libratbag-util.h"
+
+#define HOLTEK8_FW_VERSION_LEN 4
+
+enum holtek8_sensor {
+	HOTLEK8_SENSOR_UNKNOWN = 0,
+	HOTLEK8_SENSOR_PAW3333,
+	HOTLEK8_SENSOR_PMW3320,
+};
+
+struct holtek8_device_data {
+	char *fw_version;
```

Comment:
(this was probably based on the sinowealth driever but) you can probably make this an array too

Reply:
done!